seerep_srv/seerep_core_fb/src/core_fb_conversion.cpp

```diff
@@ -125,6 +125,164 @@ seerep_core_msgs::QueryTf CoreFbConversion::fromFb(const seerep::fb::TransformSt
   return queryTf;
 }
 
+seerep_core_msgs::camera_intrinsics CoreFbConversion::fromFb(const seerep::fb::CameraIntrinsics& ci)
+{
+  seerep_core_msgs::camera_intrinsics ciCore;
+
+  CoreFbConversion::fromFbDataHeader(ci.header(), ciCore.header, seerep_core_msgs::Datatype::Unknown);
```

Comment:
You added seq to the ciCore.header, but in the fb msg there is no seq. Please add seq also to the fb header msg and extent the conversion function